pyo3-build-config/src/impl_.rs

```diff
@@ -226,7 +232,7 @@ SHARED = bool(get_config_var("Py_ENABLE_SHARED"))
 print("implementation", platform.python_implementation())
 print("version_major", sys.version_info[0])
 print("version_minor", sys.version_info[1])
-print("shared", PYPY or ANACONDA or WINDOWS or FRAMEWORK or SHARED)
+print("shared", PYPY or GRAALPY or ANACONDA or WINDOWS or FRAMEWORK or SHARED)
```

Comment:
I wonder, should we be checking that the version is 24.0 or above, and bailing out if that's not the case?

Reply:
Yes, good idea. Like this? https://github.com/PyO3/pyo3/pull/3247/commits/bca1c5a305d956a085451592c889077748f511f0

Reply:
That looks like a suitable implementation to me 👍